framework/src/outputs/VariableResidualNormsDebugOutput.C

```diff
@@ -62,7 +62,8 @@ VariableResidualNormsDebugOutput::output()
   oss << "    |residual|_2 of individual variables:\n";
   for (unsigned int var_num = 0; var_num < _sys.n_vars(); var_num++)
   {
-    Real var_res_id = _sys.calculate_norm(_nl.RHS(), var_num, DISCRETE_L2);
+    NonlinearSystemBase & nl = const_cast<NonlinearSystemBase &>(_nl);
```

Comment:
ok so it did hurt, and it s why it had not been done yet
let's remove that last change